docs/dev/benchmark.md

```diff
@@ -1,3 +1,3 @@
 # Benchmark
 
-This page has been removed, please visit https://lincolnloop.github.io/python-package-manager-shootout/ for a detailed benchmark report.
+This page has been removed, please visit [new benchmark page](https://lincolnloop.github.io/python-package-manager-shootout/) for a detailed benchmark report.
```

Comment:
Why "new"?

Also maybe:

```suggestion
This page has been removed, please visit [this new benchmark page](https://lincolnloop.github.io/python-package-manager-shootout/) for a detailed benchmark report.
```

Reply:
直接使用 URL 会触发 `MD034/no-bare-urls: Bare URL used` 警告，我不知道怎么描述这个地址，我想表达的意思是 查看这个新的页面地址， 可能在语言翻译上不能很好的表达，感谢修改。
---
内容是否进行换行，是我在进行对照中文翻译的时候，个别英文语句在换行的位置没有任何的标点符号，我需要对内容进行重新换行。我不清楚原本换行是根据什么情况，是文本长度还是标点符号，最好能规范下，后面我对文档在进行更改。
---
Using the URL directly will trigger the 'MD034/no-bare-urls: Bare URL used' warning, I don't know how to describe this address, what I want to express is to view this new page address, may not be well expressed in the language translation, thanks for the modification.

---

Whether the content is broken or not, when I am translating against Chinese, there is no punctuation mark in the position of individual English sentences in the line break, and I need to re-wrap the content. I don't know what the original line break was based on, whether it was the length of the text or the punctuation, it would be better to standardize it, and then I will make changes to the document.

Reply:
Suggested link text: **Python Package Manager Shootout by Lincoln Loop**